src/Resources/views/Admin/ShippingMethod/_image.html.twig

```diff
@@ -0,0 +1,6 @@
+{% if shipping_method.image is not empty and shipping_method.image.path is not empty %}
```

Comment:
Does 'empty' in twig realize the 'empty(...)' php function? If so, I would check it different way

Reply:
Replaced with null comparision